### PR TITLE
Add instructions on how to upgrade to Rancher Prime

### PR DESCRIPTION
**Pull request by guangyee**
Also updated the screenshots to reflect reality.

### CODE REVIEW

**Review by mbelur — APPROVED** [review]
LGTM

File: content/rancher-prime/azure/contents.lr [hunk 1]
```diff
@@ -131,6 +131,17 @@ you have chosen previously.
 Please refer to the [Rancher documentation](https://ranchermanager.docs.rancher.com/)
 on how to use Rancher.
 
+After logging into Rancher Prime, you should notice the
+**Welcome to Rancher Prime** message at the top of the screen.
+
+<img src="./rancher-prime-home.png" alt="Welcome to Rancher Prime" style="border: 2px solid  gray;">
+
+However, if your Rancher Prime PAYG deployment only have **Welcome to Rancher**
```
Comment (jeremy-moffitt) [inline content/rancher-prime/azure/contents.lr]: remove `However,` start with `If your`
s/have/has/

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
small wording changes
the two *new* images both should be resized to be roughly 800 pixels wide... they're dramatically larger than the other images in use and cause a scrollbar to be added to the page

File: content/rancher-prime/azure/contents.lr (hunk 1 above)
Comment (guangyee) [reply content/rancher-prime/azure/contents.lr]: good catch!

**Review by jeremy-moffitt — APPROVED** [review]
LGTM, thanks for fixing the picture sizes and the wording

**Pull request merged**